Cargo.toml

```diff
@@ -295,11 +291,11 @@ engine = [
 ]
 default = ["dbus_enabled", "engine"]
 dbus_enabled = ["dep:dbus", "dep:dbus-tree"]
-extras = ["dep:pretty-hex"]
+extras = ["dep:pretty-hex", "test_extras"]
 min = ["dep:termios"]
 systemd_compat = ["dep:bindgen"]
 udev_scripts = ["dep:data-encoding"]
-test_extras = ["engine"]
+test_extras = []
```

Comment:
Are we leaving this for packaging?

Reply:
I can also remove it. I think there is no problem with that.

Reply:
The reasoning behind leaving it in wasn't that strong.

Reply:
The one good reason for leaving this in was not having to update the spec file. So, yes, it was for packaging convenience. If we change it, we change the spec file. But that is no big deal, really, except that it could break the Packit/TMT structure briefly. But, we can work around that.

Reply:
Actually, I don't even think it was for packaging. I'll see about removing it.